src/components/FeatureViews/Accounts/TabComponents/Schedule.js

```diff
@@ -1,5 +1,7 @@
 import { connect } from 'react-redux';
```

Comment:
Consider removing Redux from this component (and the `connect` wrapper, and associated `mapStateToProps` and `mapDispatchToProps` at the bottom) since it is not used.